src/fparser/Fortran2003.py

```diff
@@ -4177,8 +4206,12 @@ class Masked_Elsewhere_Stmt(StmtBase): # R749
     use_names = ['Mask_Expr', 'Where_Construct_Name']
     @staticmethod
     def match(string):
-        if string[:9].upper()!='ELSEWHERE': return
-        line = string[9:].lstrip()
+        if string[:9].upper()!='ELSEWHERE' and \
```

Comment:
Adds support for the keyword ELSEWHERE being written as ELSE WHERE. Good. However, would it work with more than one space between? If not, should you not test and code for this case?